sjrk.storyTelling.server.config.json5.example

```diff
@@ -0,0 +1,25 @@
+{
+    "type": "sjrk.storyTelling.server.config",
+    "options": {
+        "gradeNames": ["sjrk.storyTelling.server"],
+        "components": {
+            "server": {
+                "options": {
+                    "globalConfig": {
+                        // the HTTP port the server will hosted from, default is 8081
+                        "port": "8081",
+
+                        // the custom theme to load in on top of the base theme
+                        "theme": "learningReflections",
+                        
+                        // a default file to load at the site root (e.g. index.html)
+                        "themeIndexFile": "introduction.html",
+
+                        // if true, saving stories to the database will be enabled
```

Comment:
As you said in the readme, this flag is actually to enable/disable the authoring part. It's probably more accurate to call it "authoringEnabled".

Reply:
I think the comment in the example json5 file is actually more accurate, and the comment in the readme is descriptive of the behaviour of _both_ the server flag and the UI-side flag, which are still separately defined. I will add a comment in the readme to indicate that both should be set together and should match each other.

Reply:
Why does UI side need its own flag? /clientConfig is returning "savingsEnabled" value that is read from the server side config file.